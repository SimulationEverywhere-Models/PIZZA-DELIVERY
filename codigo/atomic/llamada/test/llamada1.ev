00:00:05:000 Llamada_in 1
00:00:15:000 Lista_ll 1
00:00:25:000 Parar_ll 1
