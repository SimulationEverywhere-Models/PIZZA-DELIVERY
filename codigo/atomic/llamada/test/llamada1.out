00:00:05:015 no_atiende 1
